components/footer.js

```diff
@@ -0,0 +1,102 @@
+import logo from "../public/logo.svg";
+import Image from "next/image";
+import { FaFacebookF } from "react-icons/fa";
+import { AiOutlineInstagram } from "react-icons/ai";
+import { AiOutlineTwitter } from "react-icons/ai";
+import { AiOutlineBehance } from "react-icons/ai";
+export default function Footer() {
+  return (
+    <>
+      <div className="text-white grid justify-items-center bg-footer lg:h-80 sm:h-auto  w-full  ">
```

Comment:
Nicely done making the footer 2 segments. However, the layout of the top and bottom aren't consistent. The icons aren't aligned perfectly with the text and logo